### ISSUE TITLE
trying to deploy repo issues

### ISSUE THREAD

**Issue by Mike-Cleaver-LB** [SalesforceLabs/Configurable-Self-Registration-for-Experience-Cloud#48]
Hi, i setup a dev org to do some more work here and actually test the code I change

using a branch from main, i seem to be missing a field on Custom_Registration_Configuration__mdt: could not find fields: Field_Show_Password_Visibility__c

can you please update the repo to add this, also can you remove RetrievePersonAccountRecordTypesTestOld from the repo as it isn't needed (it causes deployment issues)

**Comment by jlowe-SFDC** [comment]
RetrievePersonAccountRecordTypesTestOld had to remain I'm afraid due to the packaging restrictions in Salesforce. It was used in the original component and I am unable to delete it. What is missing from the branch is the actual class that these tests reference which is likely to be your failure so I've added that back in to the repo.

Field_Show_Password_Visibility__c is a deprecated field. The metadata still references it for some reason so I've updated that too.

Hopefully it is deployable now.

**Comment by jlowe-SFDC** [comment]
Actually, the best way to manage this is via the .forceignore file. That way the files can live in this repository but they will be ignored during deployment so I've made that change.

![image](https://github.com/user-attachments/assets/d90e1dd0-b904-4e8d-af3a-d3fc032ed984)

Also a couple of other things that I've spotted with reference to users that were in my test org that shouldn't be in  the repository so I've removed those too.
